src/safe_apps/admin.py

```diff
@@ -2,5 +2,36 @@
 
 from .models import SafeApp, Provider
 
-models = [SafeApp, Provider]
-admin.site.register(models)
+
+class NetworksFilter(admin.SimpleListFilter):
+    title = 'Networks'
+    parameter_name = 'networks'
+
+    def lookups(self, request, model_admin):
+        values = SafeApp.objects.values_list('networks', flat=True)
+        # lookups requires a tuple to be returned – (value, verbose value)
+        networks = [(network, network) for networks in values for network in networks]
+        networks = sorted(set(networks))
+        print(networks)
+        return networks
+
+    def queryset(self, request, queryset):
+        value = self.value()
+        if value:
```

Comment:
Since python 3.8 you can do `if value := self.value():` (just as a tip, but this way is ok and more compatible with previous versions)